packages/edge/docs/README.md

```diff
@@ -0,0 +1,264 @@
+
+## Functions
+
+### geolocation
+
+▸ **geolocation**(`request`): [`Geo`](interfaces/Geo.md)
+
+Returns the location information from for the incoming request
+
+**`See`**
+
+- [CITY_HEADER_NAME](README.md#city_header_name)
+- [COUNTRY_HEADER_NAME](README.md#country_header_name)
+- [REGION_HEADER_NAME](README.md#region_header_name)
+- [LATITUDE_HEADER_NAME](README.md#latitude_header_name)
+- [LONGITUDE_HEADER_NAME](README.md#longitude_header_name)
+
+#### Parameters
+
+| Name      | Type      | Description                                                     |
+| :-------- | :-------- | :-------------------------------------------------------------- |
+| `request` | `Request` | The incoming request object which provides the geolocation data |
+
+#### Returns
+
+[`Geo`](interfaces/Geo.md)
+
+#### Defined in
+
+[src/edge-headers.ts:76](https://github.com/vercel/vercel/blob/main/packages/edge/src/edge-headers.ts#L76)
+
+---
+
+### ipAddress
+
+▸ **ipAddress**(`request`): `string` \| `undefined`
+
+Returns the IP address of the request from the headers.
+
+**`See`**
+
+[IP_HEADER_NAME](README.md#ip_header_name)
+
+#### Parameters
+
+| Name      | Type      | Description                                       |
+| :-------- | :-------- | :------------------------------------------------ |
+| `request` | `Request` | The incoming request object which provides the IP |
+
+#### Returns
+
+`string` \| `undefined`
+
+#### Defined in
+
+[src/edge-headers.ts:62](https://github.com/vercel/vercel/blob/main/packages/edge/src/edge-headers.ts#L62)
+
+---
+
+### next
+
+▸ **next**(`init?`): `Response`
+
+Returns a Response that instructs the system to continue processing the request.
+
+**`Example`**
+
+<caption>No-op middleware</caption>
+
+```ts
+import { next } from '@vercel/edge';
+
+export default function middleware(_req: Request) {
+  return next();
+}
+```
+
+**`Example`**
+
+<caption>Add a response headers to all requests</caption>
+
+```ts
+import { next } from '@vercel/edge';
+
+export default function middleware(_req: Request) {
+  return next({
+    headers: { 'x-from-middleware': 'true' },
+  });
+}
+```
+
+#### Parameters
+
+| Name    | Type                                                   | Description                         |
+| :------ | :----------------------------------------------------- | :---------------------------------- |
+| `init?` | [`ExtraResponseInit`](interfaces/ExtraResponseInit.md) | additional options for the response |
+
+#### Returns
+
+`Response`
+
+#### Defined in
+
+[src/middleware-helpers.ts:94](https://github.com/vercel/vercel/blob/main/packages/edge/src/middleware-helpers.ts#L94)
+
+---
+
+### rewrite
+
+▸ **rewrite**(`destination`, `init?`): `Response`
+
+Returns a response that rewrites the request to a different URL.
+
+**`Example`**
+
+<caption>Rewrite all feature-flagged requests from `/:path*` to `/experimental/:path*`</caption>
```

Comment:
the markdown code tags here aren't rendering because this is inside html

Reply:
Yeah I know it, I didn't want to have the `<caption>` but seems like it is needed for `@example` 😢